parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/ProducerManager.java

```diff
@@ -262,7 +269,7 @@ protected void commitOffsets(@NonNull Map<TopicPartition, OffsetAndMetadata> off
                             // try wait again
                             commitTransaction();
                         }
-                        boolean transactionModeIsReady = lastErrorSavedForRethrow == null || !lastErrorSavedForRethrow.getMessage().contains("Invalid transition attempted from state READY to state COMMITTING_TRANSACTION");
+                        boolean transactionModeIsReady = lastErrorSavedForRethrow == null || !lastErrorSavedForRethrow.getMessage().contains("Invalid transaction attempted from state READY to state COMMITTING_TRANSACTION");
```

Comment:
Looks like it was meant to say 
```suggestion
                        boolean transactionModeIsReady = lastErrorSavedForRethrow == null || !lastErrorSavedForRethrow.getMessage().contains("Invalid transition attempted from state READY to state COMMITTING_TRANSACTION");
```

Reply:
🤦